modules/light-clients/07-tendermint/upgrade_test.go

```diff
@@ -86,6 +87,38 @@ func (suite *TendermintTestSuite) TestVerifyUpgrade() {
 			},
 			expErr: nil,
 		},
+		{
+			name: "successful upgrade scales trusting period with unbonding period decrease",
+			setup: func() {
+				newUnbondingPeriod := time.Hour * 24 * 7 * 2
```

Comment:
Lemme know if you want a dedicated test func to perform assertions on new trusting period.
This test case covers the code but doesn't do strict assertions.

Reply:
I think it would be good to assert the new trusting period is what we expect

Reply:
Agree, I'll add a separate test func for this. Will try to get around to it tomorrow or so

Reply:
added in https://github.com/cosmos/ibc-go/pull/8185/commits/bfedb3d57c53a929f86d9c66b760d8d2dd71efed